pages/linux/lid.md

```diff
@@ -0,0 +1,12 @@
+# lid
+
+> Display a user's groups or a group's users.
+> More information: <https://manned.org/lid.8>.
+
+- Show a user's groups (primary and secondary):
+
+`sudo lid {{user}}`
```

Comment:
```suggestion
- List primary and secondary groups of a specific user:

`sudo lid {{username}}`
```

There are several similar items (groups) printed, so `list` is better sounds here.